piechart/piechart.go

```diff
@@ -192,7 +190,7 @@ func (p *PieChart) Plot(c draw.Canvas, plt *plot.Plot) {
 					if p.Labels.Values.Percentage {
 						labelText += strconv.Itoa(int((v*100/totalValues)+0.5)) + "%"
 					} else {
-						labelText += strings.TrimRight(strconv.FormatFloat(v, 'f', 2, 64), "0.")
+						labelText += strconv.FormatFloat(v, 'g', 2, 64)
```

Comment:
Oops sorry about this, I didn't spot that before.

Actually, I would do this instead:
```go
strings.TrimRight(strings.TrimRight(strconv.FormatFloat(v, 'f', 2, 64), "0"), ".")
```

The point is that I would like that:
- **integer values** do not display the decimal part (hence the 2 trims, one after the other to avoid the bug on 10.00)
- **floating values** show only 2 decimal digits (hence the precision `2`)

Display mode `g` would have strange behavior on numbers like 12.3456, which would display as `12` with precision `2`, so I think keeping `f` mode and trimming manually is more appropriate.